src/proposals/Proposal.sol

```diff
@@ -124,6 +136,11 @@ abstract contract Proposal is Test, Script, IProposal {
         }
     }
 
+    /// @notice Create actions for the proposal
+    /// @dev implementations should use buildModifier modifier
+    /// TODO remove implementation once move from internal to public functions
+    function build() public virtual {}
```

Comment:
I think we should have either `build` or `_build` and not both of them.

Reply:
We are going to move the internal functions to public. I add this now so I could use on tests but I'm remove all the internal functions in a separate PR